tx_kernel/asm/copy.masm

```diff
@@ -0,0 +1,229 @@
+#! assert_eqw(A: Word, B: Word)
```

Comment:
@bobbinth I used this style on my example, but you mentioned on the review of `memcopy` that you find it confusing, let me know if I should follow the same style here.